dashboard/src/components/DeploymentFormBody/DifferentialTab.tsx

```diff
@@ -0,0 +1,48 @@
+import { CdsIcon } from "@clr/react/icon";
+import React, { useEffect, useState } from "react";
+import { DeploymentEvent } from "shared/types";
+
+interface IDifferentialSelectorProps {
+  deploymentEvent: DeploymentEvent;
+  deployedValues: string;
+  defaultValues: string;
+  appValues: string;
+}
+
+export default function DifferentialTab({
+  deploymentEvent,
+  deployedValues,
+  defaultValues,
+  appValues,
+}: IDifferentialSelectorProps) {
+  const [newChanges, setNewChanges] = useState(false);
+  const [oldValues, setOldValues] = useState("");
+  const setNewChangesFalse = () => setNewChanges(false);
+
+  useEffect(() => {
+    if (deploymentEvent === "upgrade") {
+      // If there are already some deployed values (upgrade scenario)
+      // We compare the values from the old release and the new one
```

Comment:
The code looks like it is setting the old values to the previously deployed values... ah, that's what you mean (the use of "old release" got me, thought you meant the previous chart release as opposed to the previously deployed helm release).